src/metadata/metadata_hash.c

```diff
@@ -2506,6 +2506,28 @@ static void ocf_metadata_hash_set_collision_prev(
 		ocf_metadata_error(cache);
 }
 
+static uint64_t ocf_metadata_hash_get_num_collision_pages(
+		struct ocf_cache *cache)
+{
+	struct ocf_metadata_hash_ctrl *ctrl =
+		(struct ocf_metadata_hash_ctrl *) cache->metadata.iface_priv;
+	struct ocf_metadata_raw *raw =
+		&ctrl->raw_desc[metadata_segment_collision];
+
+	return raw->ssd_pages;
+}
+
+static uint64_t ocf_metadata_hash_get_collision_page(
```

Comment:
Please make types consistent. Sometimes uint64_t is used for page id, sometimes uint32_t.

Reply:
Changed all types to uint32, this is consistent with most of our code. This requires downcasting from 64bit integer in one place in metadata_raw